src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -125,7 +125,8 @@ private Map<Dac, List<User>> groupUsersByDacs(List<Dac> dacs, List<User> allDacM
   }
 
   public List<Dac> findDacsWithMembersOption(Boolean withMembers) {
-    List<Dac> dacs = dacDAO.findAll();
+    List<Dac> dacs = dacDAO.findAll().stream().sorted(
```

Comment:
Alternatively, you could add the sort in the query is likely more efficient than here in code.